nautobot/dcim/tests/test_filters.py

```diff
@@ -1429,60 +1431,73 @@ class ManufacturerTestCase(FilterTestCases.NameSlugFilterTestCase):
     def setUpTestData(cls):
         common_test_data(cls)
 
-        devices = Device.objects.all()
-
-        manufacturers = (
-            Manufacturer.objects.create(name="Manufacturer 4", slug="manufacturer-4", description="A"),
-            Manufacturer.objects.create(name="Manufacturer 5", slug="manufacturer-5", description="B"),
-            Manufacturer.objects.create(name="Manufacturer 6", slug="manufacturer-6", description="C"),
-        )
+        devices = list(Device.objects.all()[:3])
 
-        InventoryItem.objects.create(device=devices[0], name="Inventory Item 1", manufacturer=manufacturers[0])
-        InventoryItem.objects.create(device=devices[1], name="Inventory Item 2", manufacturer=manufacturers[1])
-        InventoryItem.objects.create(device=devices[2], name="Inventory Item 3", manufacturer=manufacturers[2])
+        InventoryItem.objects.create(device=devices[0], name="Inventory Item 1", manufacturer=cls.manufacturers[0])
+        InventoryItem.objects.create(device=devices[1], name="Inventory Item 2", manufacturer=cls.manufacturers[1])
+        InventoryItem.objects.create(device=devices[2], name="Inventory Item 3", manufacturer=cls.manufacturers[2])
 
     def test_description(self):
-        params = {"description": ["A", "B"]}
-        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
+        descriptions = [m.description for m in self.manufacturers[:2]]
+        params = {"description": descriptions}
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), len(descriptions))
```

Comment:
```suggestion
        self.assertEqual(self.filterset(params, self.queryset).qs.count(), self.queryset.filter(description__in=descriptions).count())
```

Reply:
Same throughout these tests

Reply:
Still won't work because of the newlines in the descriptions, try as I might. So I moved it back to having the factory use `sentence` instead here. At least for now.